README.md

```diff
@@ -1,64 +1,59 @@
-3. [Status](#2-status)
-	* [Device Model](#device-model)
-	* [TODO](#todo)
-4. [rust-vmm dependency](#4-rust-vmm-dependency)
-	* [Firecracker and crosvm](#firecracker-and-crosvm)
-5. [Community](#5-community)
-	* [Contribute](#contribute)
-	* [Join us](#join-us)
-6. [Security](#6-security)
+- [1. What is Cloud Hypervisor?](#1-what-is-cloud-hypervisor)
+	- [Objectives](#objectives)
+		- [High Level](#high-level)
+		- [Architectures](#architectures)
+		- [Guest OS](#guest-os)
+- [2. Getting Started](#2-getting-started)
+	- [Clone and build](#clone-and-build)
+		- [Containerized builds and tests](#containerized-builds-and-tests)
+	- [Run](#run)
+		- [Cloud image](#cloud-image)
+		- [Custom kernel and disk image](#custom-kernel-and-disk-image)
+			- [Building your kernel](#building-your-kernel)
+			- [Disk image](#disk-image)
+			- [Booting the guest VM](#booting-the-guest-vm)
+- [3. Status](#3-status)
+	- [Hot Plug](#hot-plug)
+	- [Device Model](#device-model)
+	- [TODO](#todo)
+- [4. `rust-vmm` project dependency](#4-rust-vmm-project-dependency)
+	- [Firecracker and crosvm](#firecracker-and-crosvm)
+- [5. Community](#5-community)
+	- [Contribute](#contribute)
+	- [Join us](#join-us)
 
 # 1. What is Cloud Hypervisor?
 
-**This project is an experiment and should not be used with production workloads.**
-
 Cloud Hypervisor is an open source Virtual Machine Monitor (VMM) that runs on top of [KVM](https://www.kernel.org/doc/Documentation/virtual/kvm/api.txt).
 The project focuses on exclusively running modern, cloud workloads, on top of a limited set of hardware architectures and platforms.
-Cloud workloads refers to those that are usually run by customers inside a cloud provider. For our purposes this means modern
-Linux* distributions with most I/O handled by paravirtualised devices (i.e. virtio), no requirement for legacy devices and recent CPUs and KVM.
+Cloud workloads refers to those that are usually run by customers inside a cloud provider. For our purposes this means modern operating systems with most I/O handled by paravirtualised devices (i.e. virtio), no requirement for legacy devices, and 64-bit CPUs.
 
 Cloud Hypervisor is implemented in [Rust](https://www.rust-lang.org/) and is based on the [rust-vmm](https://github.com/rust-vmm) crates.
 
 ## Objectives
 
 ### High Level
 
-* KVM and KVM only based
+* KVM based
 * Minimal emulation
 * Low latency
 * Low memory footprint
 * Low complexity
 * High performance
 * Small attack surface
 * 64-bit support only
-* Build time configurable CPU, memory, PCI and NVDIMM hotplug
+* CPU, memory, PCI hotplug
 * Machine to machine migration
 
 ### Architectures
 
-`cloud-hypervisor` only supports the `x86-64` CPU architecture for now.
-
-We're planning to add support for the `AArch64` architecture in the future.
+Cloud Hypervisor supports the `x86-64` and `AArch64` architecture. There are some small differences in functionality between the two architectures.
```

Comment:
Are those differences noted anywhere? If not, would it make sense to add the broad strokes here?

Reply:
I referenced the github tracking issue.